src/components/Book/booking/booking.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './booking.css';
+
+function Booking() {
+  return (
+    <h2 id="construction">Under construction</h2>
+  );
+}
```

Comment:
- [ ] Kindly render a `button` with the text `Check Status` and not an `h2` element with the text `Under Construction`. 👍